src/data/teams.js

```diff
@@ -49,11 +49,11 @@ const teams = [
         technologies: ["HTML/CSS", "Jekyll/Liquid", "React"],
         projects: ["Bruin Quest Website", "Learning Labs"],
     },
-    // {
-    //     logotype: studio,
-    //     technologies: ["HTML/CSS", "Jekyll/Liquid", "React"],
-    //     projects: ["Static Site", "Learning Labs"],
-    // },
+    {
+         logotype: studio,
+         technologies: ["HTML/CSS", "Jekyll/Liquid", "React"],
+         projects: ["Static Site", "Learning Labs"],
```

Comment:
The technologies and projects sections are just remnants of copy paste, we should try to reach out to someone at Studio (or check their website) to see if we can find out any of their current projects!

For technologies, I would assume Unity (?). Though it might be worth asking about that too.